x-pack/test/cloud_security_posture_functional/page_objects/rule_page.ts

```diff
@@ -52,11 +52,13 @@ export function RulePagePageProvider({ getService, getPageObjects }: FtrProvider
     toggleBulkActionButton: async () => {
       const bulkActionButtonToBeClicked = await testSubjects.find(RULES_BULK_ACTION_BUTTON);
       await bulkActionButtonToBeClicked.click();
+      await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
What I don't understand is why we need to fail till loading is finished if the action doesn't cause any network call and I guess no loading should happen. My concern is that if we don't understand why, the fix works just by luck. Can it be that we need to wait for loading after `await rule.navigateToRulePage('cis_k8s', '1.0.1');` in `beforeEach` ?

Reply:
I also tried to just unskip the test suit and run the flaky test runner more or less the same number of runs as you did in https://github.com/elastic/kibana/pull/176139 and none of the runs failed.

Reply:
regarding your first comment, I'm pretty sure that we already wait for loading after navigating to the Rule Page (the waitForLoadingHasFinished is called within the navigateToRulePage function) prior to this PR 
but it still fail